test/testlocal/nft/anvil/deployment/AnvilContracts.t.sol

```diff
@@ -5,6 +5,7 @@ abstract contract AnvilContracts {
   address public ChainlinkRelayerFactory_Address = 0x5FbDB2315678afecb367f032d93F642f64180aa3;
   address public DenominatedOracleFactory_Address = 0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512;
   address public DelayedOracleFactory_Address = 0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0;
+  address public InternalCallsExtension_Address = 0x0DCd1Bf9A1b36cE34237eEaFef220932846BCD82;
```

Comment:
Whats the purpose of this line?

Reply:
It was added when I deployed the contracts on anvil as part of the InternalCallsWatcher.sol contract.  Seems like it should have been there before?